pages/api/models/index.ts

```diff
@@ -64,8 +64,8 @@ const associations = ({
 }: any) => {
     PackagedService.belongsTo(Service, { foreignKey: 'service_id' })
     PackagedService.belongsTo(ServicePackage, { foreignKey: 'service_package_id' })
-    ProvidedService.belongsTo(Service, { foreignKey: 'service_id' })
-    ProvidedService.belongsTo(Project, { foreignKey: 'project_id' })
+    Project.hasMany(ProvidedService, { foreignKey: 'project_id' })
+    Service.hasMany(ProvidedService, { foreignKey: 'service_id' })
```

Comment:
Does this change affect the schema?

Reply:
Noo, `belongsTo` didn't let me use the connection from Project/Service to ProvidedService